src/components/button-bar/button-bar.test.js

```diff
@@ -77,4 +79,65 @@ context("Test for Button-Bar component", () => {
       cy.checkAccessibility();
     });
   });
+
+  describe("check ButtonBar can be navigated using a keyboard", () => {
+    it("should verify ButtonBar with wrapped components can be navigated using keyboard", () => {
+      CypressMountWithProviders(<ButtonBarCustom />);
+
+      buttonDataComponent().eq(0).focus();
+      buttonDataComponent().eq(0).tab();
+      buttonDataComponent().eq(1).should("be.focused");
+      buttonDataComponent().eq(0).should("not.be.focused");
+      buttonDataComponent().eq(1).tab();
+      buttonDataComponent().eq(2).should("be.focused");
+      buttonDataComponent().eq(1).should("not.be.focused");
+    });
+  });
+
+  describe("when custom Button wrapper components are used as children in ButtonBar", () => {
+    it("Button size is small / 81px when the size prop is set to small and passed to ButtonBar", () => {
+      CypressMountWithProviders(<ButtonBarWithWrapper size="small" />);
+
+      buttonDataComponent().then(($el) => {
+        useJQueryCssValueAndAssert($el, "width", 81);
+      });
+    });
+
+    it("Button is fullWidth / 339px when the fullWidth prop is passed to ButtonBar", () => {
+      CypressMountWithProviders(<ButtonBarWithWrapper fullWidth />);
```

Comment:
May I know why we are checking fullwidth prop again as we have a test case for fullwidth in line number 43 ?